examples/shell/esp32/main/main.cpp

```diff
@@ -36,6 +36,10 @@ extern "C" void app_main(void)
     chip::Platform::MemoryInit();
     chip::DeviceLayer::PlatformMgr().InitChipStack();
     chip::DeviceLayer::PlatformMgr().StartEventLoopTask();
+
+    int ret = Engine::Root().Init();
+    assert(ret == 0);
```

Comment:
VerifyOrDie maybe? I think assert may be removed in release builds and we seem to actually want some reasonable error here (boot loop is not that reasonable, but I expect the verify to maybe print something out)